cmd/vicadmin/vicadm.go

```diff
@@ -46,15 +46,20 @@ import (
 	"github.com/vmware/govmomi/vim25/types"
 
 	"github.com/vmware/vic/lib/metadata"
+	"github.com/vmware/vic/lib/pprof"
 	"github.com/vmware/vic/pkg/trace"
 	"github.com/vmware/vic/pkg/vsphere/compute"
 	"github.com/vmware/vic/pkg/vsphere/extraconfig"
 	"github.com/vmware/vic/pkg/vsphere/session"
 	"github.com/vmware/vic/pkg/vsphere/vm"
 )
 
+const (
+	timeout = time.Duration(2 * time.Second)
```

Comment:
This is internal to the appliance so network latency not an issue - we end up blocking with log bundle collection for `components * timeout` seconds and to keep it responsive I'd prefer to have this reasonably low.
Wish there was a way to detect nested to automatically x10 but no.

We can bump it to 5 to allow for massive CPU ready time in nested envs - but would prefer to externalize it eventually. We just don't have a structured place to put it for now (I don't want to just keep dumping finely granular config into the metadata/VirtualContainerHostConfigSpec without more structure as it'll be awkward to change.